integration/nevermined/Bookmarks.test.ts

```diff
@@ -90,18 +87,16 @@ describe('Bookmarks', () => {
     it('should delete a bookmark by id', async () => {
         await nevermined.bookmarks.deleteOneById(id)
 
-        setTimeout(async () => {
-            const response = await nevermined.bookmarks.findManyByUserId(
-                newBookmark.userId
-            )
-
-            /* eslint-disable @typescript-eslint/camelcase */
-            assert.deepEqual(response, {
-                page: 1,
-                total_pages: 1,
-                total_results: 1,
-                results: []
-            })
-        }, 2000)
+        await sleep(1000)
```

Comment:
We need to check from the marketplace api side why do we need to wait for elasticsearch/marketplace-api before the operation is actually reflected. This shouldn't happen especially if you are reusing the same connection. I wonder if there is some chaching happening.

This is okay for now but in the future will lead to tests just randomly failing. I have the same problem in the sdk

Reply:
the problem is elasticsearch, it doesn't make persistent in the database immediately, I need to figure out how can I do immediately persistent without call to refresh directly in the database, after I have that, I will update this